### DISCUSSION TITLE
Consolidate 1 "Projects" section and 1 "Discussions" section for all projects

### DISCUSSION THREAD

**Discussion by eddybrando** [Qiskit/qiskit.org#2340] [Ideas]
Each GitHub repository can include "Projects" and "Discussion".

Since we have multiple repositories, we can also have multiple "Projects" and "Discussion" sections. This is the current situation.

This proposal is to consider having only 1 "Projects" section and 1 "Discussion" section, to consolidate all planing and discussions in one central place.

These would be the ones in this repository (https://github.com/Qiskit/qiskit.org).

In all other repositories, we would then need to disable the "Projects" and "Discussion" options in the repository settings, and also ideally point to the central "Projects" and "Discussion" links (in the README, for example).

---

The repositories that would be included in this change:

- https://github.com/Qiskit/qiskit.org
- https://github.com/qiskit-community/platypus
- https://github.com/qiskit-community/qiskit-vue
- https://github.com/qiskit-community/qiskit-web-components

**Comment by abdonrd** [comment]
For me it makes sense to have a single Project, without a doubt; in the end it is how we work in the team.

I'm not so sure about the discussions. I think it does make sense that each discussion happens in its own repository, just like the issues.

**Reply by eddybrando** [reply]
That's a very valid point.

I think there is no objection for the single **Project**, but indeed one single **Discussion** might make external contributors life harder.

**Comment by abdonrd** [comment]
I don't know if this is a best place, but maybe we could use the team discussions?

Now we have teams here:

- [https://github.com/orgs/Qiskit/teams/**qiskit-digital**/discussions](https://github.com/orgs/Qiskit/teams/qiskit-digital/discussions)
- [https://github.com/orgs/Qiskit/teams/**qiskit-digital-devs**/discussions](https://github.com/orgs/Qiskit/teams/qiskit-digital-devs/discussions)

**Comment by eddybrando** [comment]
### Conclusion

We'll have one central **Projects** section in https://github.com/Qiskit/qiskit.org/projects, were we will track the issues and progress of all of our repositories.

Additionally, each repository will have its own **Discussions** section for discussions that are scoped and really only relevant to that project.

**Reply by abdonrd** [reply]
Seems that soon we will be able to have projects in the Qiskit org! https://github.com/Qiskit/qiskit.org/issues/2363
